.gitignore

```diff
@@ -21,6 +21,8 @@ vendor/ruby/
 .ruby-version
 .ruby-gemset
 .byebug_history
+.pry_history
+jetty/
```

Comment:
Why is this here?

Reply:
b/c i use pry.  I dunno how jetty got there.

Reply:
haha - jetty dir on my laptop is from 2017.  I can remove from .gitignore if you like.